packages/ods-react/src/components/combobox/src/components/combobox-content/ComboboxContent.tsx

```diff
@@ -0,0 +1,76 @@
+import { Combobox as VendorCombobox, useComboboxContext } from '@ark-ui/react/combobox';
+import { Portal } from '@ark-ui/react/portal';
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX, forwardRef, useContext, useEffect, useRef } from 'react';
+import { type ComboboxCustomGroupRendererArg, type ComboboxGroupItem, type ComboboxOptionItem, useCombobox } from '../../context/combobox';
+import { ComboboxContext } from '../../context/combobox';
+import { isGroup } from '../../controller/combobox';
+import { ComboboxGroup } from '../combobox-group/ComboboxGroup';
+import { ComboboxOption } from '../combobox-option/ComboboxOption';
+import { type ComboboxCustomOptionRendererArg } from '../combobox-option/ComboboxOption';
+import style from './comboboxContent.module.scss';
+
+interface ComboboxContentProp extends ComponentPropsWithRef<'div'> {
+  customGroupRenderer?: (arg: ComboboxCustomGroupRendererArg) => JSX.Element;
+  customOptionRenderer?: (arg: ComboboxCustomOptionRendererArg) => JSX.Element;
+}
+
+const ComboboxContent: FC<ComboboxContentProp> = forwardRef(({
+  className,
+  customGroupRenderer,
+  customOptionRenderer,
+  ...props
+}, ref): JSX.Element => {
+  const { collection } = useComboboxContext();
+  const { filteredItems } = useCombobox();
+  const { setPlacement } = useContext(ComboboxContext);
+  const localRef = useRef<HTMLDivElement>(null);
+  const contentRef = (ref as React.RefObject<HTMLDivElement>) || localRef;
+
+  useEffect(() => {
```

Comment:
why do you handle all this positionning stuff manually, Ark Content are positionned automatically already